plugins/source/k8s/resources/services/apps/deployments.go

```diff
@@ -1,363 +1,161 @@
+// Code generated by codegen; DO NOT EDIT.
+
 package apps
 
 import (
-	"context"
-	"encoding/json"
-
 	"github.com/cloudquery/cloudquery/plugins/source/k8s/client"
-	"github.com/cloudquery/cq-provider-sdk/provider/diag"
-	"github.com/cloudquery/cq-provider-sdk/provider/schema"
-	appsv1 "k8s.io/api/apps/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/cloudquery/plugin-sdk/schema"
 )
 
 func Deployments() *schema.Table {
 	return &schema.Table{
-		Name:         "k8s_apps_deployments",
-		Description:  "Deployment enables declarative updates for Pods and ReplicaSets.",
-		Resolver:     fetchAppsDeployments,
-		Multiplex:    client.ContextMultiplex,
-		DeleteFilter: client.DeleteContextFilter,
-		IgnoreError:  client.IgnoreForbiddenNotFound,
-		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"uid"}},
+		Name:      "k8s_apps_deployments",
+		Resolver:  fetchAppsDeployments,
+		Multiplex: client.ContextMultiplex,
 		Columns: []schema.Column{
```

Comment:
OOps, I forgot `commonContextField`, so users won't be able to know which resource is from which context... Need to re-add this.